orderable/managers.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class OrderableManager(models.Manager):
+
+    def before(self, orderable):
+        return self.filter(sort_order__lt=orderable.sort_order).last()
+
+    def after(self, orderable):
+        return self.filter(sort_order__gt=orderable.sort_order).first()
```

Comment:
I've done a small tidy of getting the filtered object. The related manager already deals with most of this from the queryset.

Otherwise you need to do this to get round it:

`ordered_instance.__class__.objects.after(ordered_instance)`

And frankly, if you want to do that, you are welcome to it.